openslides_backend/action/action.py

```diff
@@ -181,6 +183,12 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         """
         return instance
 
+    def final_update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Updates one instance of the payload or executes other actions.
+        """
+        return instance
```

Comment:
I'm not a fan of this. It seems hacky and bypasses (by intent, but still) the relation logic. With the automatic inclusion of all updated models in the `additional_relation_models`, the old version should have worked. What was the error there when just executing `MeetingReplaceProjectorId` as a dependency?

Reply:
(Note: I think @r-peschke did this part of the code)

Reply:
Ah, I see. @r-peschke did you change this after #486 was merged? If so, what exactly did not work with the old setup?

Reply:
The problem is the update-sequence of the renaming-action. There is no way of putting them behind the deletions of the relations, generated by the basic delete action. The only thing we could do, is prevent the delete logic of resetting relations, changed in between. But IMO to do this correctly with all kind of relations is a harder way to go.

Reply:
I think with #518 this should work. Example: Let's say `meeting/1/default_projector_$motion_id=1`, `meeting/1/reference_projector_id=42`  and we delete `projector/1`. `ProjectorDelete` calls `MeetingReplaceProjectorId` __before__ any write requests are created and, more importantly, before any changes are written to the cache layer. `MeetingReplaceProjectorId` sets `meeting/1/default_projector_$motion_id=42`, which is saved to the cache and the write request is created. Then, control is returned to `ProjectorDelete`, which proceeds to delete `projector/1`, but since the relation in the meeting is already updated to another projector, it does **not** set this to `null`. Everything should work as intended.

This should probably already work with our current setup, we would just have to use the `fetch_model` method instead of `get` in the `MeetingReplaceProjectorId` action. This line seems much cleaner than adding another helper method ;)